MAUI/DataGrid/performance.md

```diff
@@ -60,9 +60,9 @@ datagrid.EnableDataVirtualization = true;
 
 The DataGrid supports loading data incrementally using the `ISupportIncrementalLoading` interface. This interface includes the `LoadMoreItemsAsync` method, which helps to load data incrementally. The `LoadMoreItemsAsync` method is called on-demand while scrolling, based on the `HasMoreItems` property.
```

Comment:
Include API reference for **ISupportIncrementalLoading**, **LoadMoreItemsAsync** and **HasMoreItems** properties.